### PR TITLE
Bug 1273413 - [TV][Home] Navigation in card selection page doesn't su…

### PR DESCRIPTION
**Pull request by luke-chang**
…pport to wrap around in both directions

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=28f2968b837d3334d655d2d6bc06008de83b41ee

File: tv_apps/smart-home/js/card_picker.js [hunk 1]
```diff
@@ -248,6 +250,18 @@
       function appendToGridView(appCardElem) {
         that.gridView.appendChild(appCardElem);
         that.appCardElems.push(appCardElem);
+
+        if (top >= 0) {
```
Comment (rexboy7) [inline tv_apps/smart-home/js/card_picker.js]: The logic here is not descriptive enough, including the meaning of `top = -1` and variable name of `cr`.

Comment (rexboy7) [inline tv_apps/smart-home/js/card_picker.js]: I would prefer to move this paragraph out of `appendToGridView`, and let it be an independent for-loop. After you found the first row you can just break the for-loop. I think that should be more easier to read.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=15c42f2c7f006d5eb9004d1222fb170a87cc7c45

**Pull request merged**